content/en/continuous-deployment/plugin-guide/policy-engine/_index.md

```diff
@@ -0,0 +1,338 @@
+metadata:
+  name: opa
+  namespace: opa # Change this to the namespace OPA is installed in
+spec:
+  selector:
+    app: opa
+  ports:
+  - protocol: TCP
+    port: 8181
+    targetPort: 8181
+</pre></code>
+</details>
+
+## Enable the Policy Engine plugin
+
+You have three options for enabling the Policy Engine plugin:
+
+1. [Armory Operator or Spinnaker Operator](#armory-operator-or-spinnaker-operator)
+1. [Spinnaker services local files](#spinnaker-services-local-files) (Spinnaker only)
+1. [Halyard](#halyard) (Spinnaker only)
+
+### Armory Operator or Spinnaker Operator              
+
+You can enable the Policy Engine plugin using the Armory Operator or the Spinnaker Operator and the sample manifest, which is in the [spinnaker-kustomize-patches repository](https://github.com/armory/spinnaker-kustomize-patches/blob/master/armory/patch-policy-engine-plugin.yml).
+
+* The sample manifest is for the Armory Operator and Armory CD. If you are using the Spinnaker Operator and Spinnaker, you must replace the `apiVersion` value "spinnaker.armory.io/" with "spinnaker.io/". For example:
+
+  * Armory Operator: `apiVersion: spinnaker.armory.io/v1alpha2`
+  * Spinnaker Operator: `apiVersion: spinnaker.io/v1alpha2`
+
+* In the `gate.spinnaker.extensibility.deck-proxy.plugins.Armory.PolicyEngine.version` entry, make sure to replace the version number listed after `&version` with the version of the plugin you want to use. For example, if you are using Armory CD 2.27.x (Spinnaker 1.27.x), you would use version 0.2.1:
+
+   {{< prism lang="yaml" line="12" >}}
+   gate:
+     spinnaker:
+        extensibility:
+         plugins:
+           Armory.PolicyEngine:
+              enabled: true
+         deck-proxy:
+           enabled: true
+          plugins:
+             Armory.PolicyEngine:
+                enabled: true
+                version: &version 0.2.1
+   {{< /prism>}}
+
+<details><summary>Show the manifest</summary>
+{{< github repo="armory/spinnaker-kustomize-patches" file="/armory/patch-policy-engine-plugin.yml" lang="yaml" options="" >}}
+</details>
+
+#### Optional settings
+##### Timeout settings
+
+You can configure the amount of time that the Policy Engine waits for a response from your OPA server. If you have network or latency issues, increasing the timeout can make Policy Engine more resilient. Use the following config to set the timeout in seconds: `spec.spinnakerConfig.profiles.spinnaker.armory.policyEngine.opa.timeoutSeconds`. The default timeout is 10 seconds if you omit the config.
+
+##### JSON validation
+
+You can configure strict JSON validation as a boolean in `spec.spinnakerConfig.profiles.dinghy.jsonValidationDisabled`:
+
+
+{{< prism lang="yaml" >}}
+spec:
+  spinnakerConfig:
+    profiles:
+      dinghy:
+        jsonValidationDisabled: <boolean>
+{{< /prism >}}
+
+The config is optional. If omitted, strict validation is on by default. When strict validation is on, existing pipelines may fail if any JSON is invalid.
```

Comment:
@dogonthehorizon is this JSON validation section valid for PE? It's a dinghy key???

Reply:
No this section is only valid for pipelines-as-code